opentreemap/treemap/templates/treemap/map_feature_photo_detail.html

```diff
@@ -0,0 +1,36 @@
+{% extends "instance_base.html" %}
+{% load threadedcomments_tags %}
+{% load instance_config %}
+{% load i18n %}
+{% load l10n %}
+
+{% block head_extra %}
+{# Facebook & Google+ scrape these properties for their share dialogs #}
+<meta property="og:url" content="{{ photo.absolute_detail_url }}" />
+<meta property="og:title" content="Map Feature Photo" />
+<meta property="og:description" content="Photo uploaded by at {{ photo.raw.created_at }}" />
```

Comment:
`og:title` and `og:description` both need i18n.  Also, we need to format the date, like so: `{{ photo.raw.created_at|date:request.instance.short_date_format }}`